packages/syft/tests/syft/lib/allowlist_test.json

```diff
@@ -3420,7 +3420,8 @@
       },
       "_version": {
         "profile": "tensor_property_noinput",
-        "return_type": "syft.lib.python.Int"
+        "return_type": "syft.lib.python.Int",
+        "deterministic": false
```

Comment:
Q: I think this issue occured when we merged dp_tensor_optim branch, could any change in that merge affected this test?